From bb637f9e214fef4b1b4316cacc74807b3d9031fc Mon Sep 17 00:00:00 2001
From: Haim Boot <hayim@marvell.com>
Date: Sun, 11 Dec 2016 15:12:56 +0200
Subject: [PATCH 1174/2241] dts: rtc: a70x0: a80x0: enable rtc interrupts

the interrupt is connected from MSS to GIC via ICU ID 77.

Change-Id: Ic69934c7ee57c5e244cce0ed6e55025d7b6c0fc7
Signed-off-by: Haim Boot <hayim@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/34565
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi | 3 +++
 arch/arm64/boot/dts/marvell/armada-cp110.dtsi   | 3 +++
 2 files changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
index 524e761..5307719 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
@@ -22,6 +22,9 @@ rtc@284000 {
 	compatible = "marvell,armada8k-rtc";
 	reg = <0x284000 0x20>, <0x284080 0x24>;
 	reg-names = "rtc", "rtc-soc";
+	#address-cells = <1>;
+	#size-cells = <0>;
+	interrupts = <ICU_GRP_NSR 77 IRQ_TYPE_LEVEL_HIGH>;
 	status = "disabled";
 };
 
diff --git a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
index a53b49f..61a0446 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
@@ -54,6 +54,9 @@ rtc@284000 {
 	compatible = "marvell,armada8k-rtc";
 	reg = <0x284000 0x20>, <0x284080 0x24>;
 	reg-names = "rtc", "rtc-soc";
+	#address-cells = <1>;
+	#size-cells = <0>;
+	interrupts = <ICU_GRP_NSR 77 IRQ_TYPE_LEVEL_HIGH>;
 	status = "disabled";
 };
 
-- 
2.7.4

